autoload/neomake/makers/ft/scala.vim

```diff
@@ -1,11 +1,23 @@
 " vim: ts=4 sw=4 et
 function! neomake#makers#ft#scala#EnabledMakers()
-    let makers = ['scalac']
-    if neomake#utils#Exists('scalastyle')
+    let makers = get(g:, 'neomake_scala_fast', 0) ? ['fsc'] : ['scalac']
```

Comment:
Instead of adding a configuration variable, how about something like this:

```
if neomake#utils#Exists('fsc')
    let makers = ['fsc']
else
    let makers = ['scalac']
endif
```

If this is not to a user's liking, they always have the ability to set the enabled makers themselves (see other comment).